### PR TITLE
Add aggregations export button

### LINKED ISSUE #1257: Nicer error message when data exports are not ready
Right now you get an XML error page if your data exports aren't ready. But I wouldn't know what to make of that if I were an average user. 

Also, exports are relatively slow. My export with 16,000 subjects took ~30 minutes. So it might be better to suggest people might need to wait "several minutes" instead of "a few seconds".

### PR DESCRIPTION
**Pull request by edpaget**
Also add ready state to buttons to indicate new data is being generated,
closes #1257

### CODE REVIEW

**Comment by edpaget** [comment]
Depends on zooniverse/Panoptes#1267 & zooniverse/aggregations#13

**Comment by chrissnyder** [comment]
Both dependencies are merged now. Is this able to be merged now? Will take a peek tomorrow if so.

**Comment by edpaget** [comment]
No I'd like to give it a try on staging. I'll post back here when it's good to go.

**Comment by camallen** [comment]
@edpaget can we get this rebased and only show up if an admin user? looks like we're ready on the aggregation engine.

**Comment by edpaget** [comment]
Will do.

**Comment by edpaget** [comment]
Should be good-to-go @camallen

File: app/partials/data-export-button.cjsx [hunk 1]
```diff
@@ -16,13 +19,19 @@ module.exports = React.createClass
...
-    apiClient.post @props.project._getURL(@props.exportType), media: content_type: 'text/csv'
+    apiClient.post @props.project._getURL(@props.exportType), media: content_type: @props.contentType
       .then =>
         @_exportsGet = null
         @setState exportRequested: true
       .catch (error) =>
         @setState exportError: error
 
+  recentAndReady: (exported) ->
+    exported? and (exported.metadata.state is 'ready' or not exported.metadata.state?)
```
Comment (camallen) [inline app/partials/data-export-button.cjsx]: can this read, `exported? and (exported.metadata.state in [ 'ready', 'finished' ] or not exported.metadata.state?)` API has the finished state currently.

Comment (edpaget) [inline app/partials/data-export-button.cjsx]: Can I just change it to 'finished' then?

Comment (camallen) [inline app/partials/data-export-button.cjsx]: for sure!

Comment (edpaget) [inline app/partials/data-export-button.cjsx]: donezo.

Comment (edpaget) [inline app/partials/data-export-button.cjsx]: undonezo!

File: app/pages/lab/data-dumps.cjsx [hunk 2]
```diff
@@ -56,6 +58,14 @@ module.exports = React.createClass
               buttonKey="projectDetails.workflowContentsExport"
               exportType="workflow_contents_export"  />
           </p>
+          {if isAdmin()
+             <p>
+              <DataExportButton
+                project={@props.project}
+                buttonKey="projectDetails.aggregationExport"
+                contextType="application/x-gzip"
```
Comment (camallen) [inline app/pages/lab/data-dumps.cjsx]: contentType?

Comment (edpaget) [inline app/pages/lab/data-dumps.cjsx]: Whoops.

**Pull request merged**